src/api/app/controllers/webui/staging_workflows/staging_projects_controller.rb

```diff
@@ -14,6 +14,7 @@ def create
         staging_project = @staging_workflow.staging_projects.build(name: staging_project_name)
 
         if staging_project.valid? && staging_project.store
+          @staging_workflow.write_to_backend
```

Comment:
Why don't you do this in a callback?

Reply:
let's see :smile_cat:

Reply:
I would really encourage putting it in the model - we had the problem with attributes being stored only from the API controller and it created quickly a mess, as the webui controllers were refactored (not to use API anymore).

Doing it in the model created different problems - especially with cassettes, but I leave you that sad story for a rainy night